workflows/prognostic_c48_run/runtime/segmented_run/cli.py

```diff
@@ -51,20 +51,32 @@ def create(url: str, fv3config_path: str):
 
 
 @cli.command()
+@click.option(
+    "--mpi_launcher",
+    default="mpirun",
+    help="MPI launcher used to run the simulation.\
+          Supported values are 'mpirun' and 'srun'.",
```

Comment:
If a string needs to be split over multiple lines, I recommend using this syntax instead.  I think using `"\"` may inadvertently include the leading spaces at the start of the second line.
```suggestion
    help="MPI launcher used to run the simulation. "
         "Supported values are 'mpirun' and 'srun'.",
```
Since the same string is used farther down in the `run-native` command definition, you could consider defining it as a module-level constant.